lib/openstudio/lib/openstudio_aws_wrapper.rb

```diff
@@ -220,13 +248,14 @@ def stop_instances(ids)
   end
 
   def terminate_instances(ids)
+    resp = nil
     begin
       resp = @aws.terminate_instances(
         instance_ids: ids
       )
     rescue Aws::EC2::Errors::InvalidInstanceIDNotFound
       # Log that the instances couldn't be found?
-      return resp = { error: 'instances could not be found' }
+      resp = { error: 'instances could not be found' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.